doc/api/deprecations.md

```diff
@@ -2655,7 +2655,7 @@ API.
 ### DEP0XXX: Use `request.destroy()` instead of `request.abort()`
```

Comment:
This (and the anchor tag on the immediate preceding line above needs a number (on master as well?).

Reply:
Yes, that should have happened while landing ce1af77f2d on master. Do you want to open a PR? Otherwise I can do that.

Reply:
Already on it 😄.

Reply:
https://github.com/nodejs/node/pull/33109